### DISCUSSION TITLE
API call works for some courts and not others

### DISCUSSION THREAD

**Discussion by warrenp94** [freelawproject/courtlistener#2634] [Q&A] [answered]
I have run a query via the API for some jurisdictions (e.g. scotus, ca1, ca2, etc):

https://www.courtlistener.com/api/rest/v3/opinions/?date_created__gte=2021-01-01T00:00:00Z&date_created__lte=2021-6-30T00:00:00Z&cluster__docket__court=ca1&cluster__precedential_status=Published

This seems to return 173 proper results.

Using the same reasoning, I have tried to pull data for e.g. mtd  with similar date filters and I get 0 results.  If I remove the date filter, I get 1770 results but I cannot find how to filter for a date range.  If I use the website, I can see that there should be results for a particular timeframe but I am not able to figure out why this query that works for some courts will not work for mtd (or various other lower courts I have tried)

I have looked at the graphic representation of the database as well as the test cases but this did not provide me with meaningful insight.  Can anyone provide a sample api call that might better illustrate how to get opinions for (e.g.) mtd for a particular date range?

**Comment by mlissner** [comment, marked as answer]
This query seems to be the one that returns 1770 results for mtd:

https://www.courtlistener.com/api/rest/v3/opinions/?cluster__docket__court=mtd&cluster__precedential_status=Published

I think your question is why the date_created filters don't work on it? 

They do seem to work to me. I used this query:

https://www.courtlistener.com/api/rest/v3/opinions/?cluster__docket__court=mtd&cluster__precedential_status=Published&date_created__gt=2022-12-14T15:21:01.934158-08:00

To get back three results. I created that query by looking at the date of the first result above, copy/pasting it into the URL bar, and then subtracting one day from it. 

My guess is that you're confusing the `date_created` field with the `date_filed` field?

**Reply by warrenp94** [reply]
Thanks for the reply.  I have no idea why I was having such difficulty with things but indeed you are right on your findings.  Sorry for wasting your time.  I looked at this several different ways and for some reason, it just wasn't working.  But with your suggestions (and a good night sleep) I seem to be fine now!  I appreciate it!
